package.json

```diff
@@ -28,6 +28,7 @@
   },
   "devDependencies": {
     "chai": "^1.10.0",
+    "md5": "^2.0.0",
```

Comment:
should this be in 'dependencies' rather than 'devDependencies'? Is the 'sha-1' dependency still needed?